src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/jpa/OffenderVisitBalance.kt

```diff
@@ -39,7 +39,7 @@ data class OffenderVisitBalance(
     return offenderBookingId == other.offenderBookingId
   }
 
-  override fun hashCode(): Int = offenderBookingId.hashCode()
+  override fun hashCode(): Int = javaClass.hashCode()
 
```

Comment:
interestingly this key not generated but use the bookingId which itself is generated, so this can change during creation